JiraReporter/Services/AuthorLoader.cs

```diff
@@ -274,7 +359,7 @@ private void SetAuthorDayLogs()
             foreach (var day in _options.ReportDates)
                 _currentAuthor.DayLogs.Add(DayLogLoader.CreateDayLog(_currentAuthor, day, _context));
             _currentAuthor.DayLogs = _currentAuthor.DayLogs.OrderBy(d => d.Date).ToList();
-            _currentAuthor.DayLogs.RemoveAll(d => d.Commits.Count == 0 && d.Issues == null);
+            _currentAuthor.DayLogs.RemoveAll(d => d.Commits.Count == 0 && (d.Issues == null || d.Issues.Count == 0));
```

Comment:
You can create a helper method for collections `IsEmpty` to check for both null and `count = 0`.